model/src/main/java/io/github/droidkaigi/feeder/Locale.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.droidkaigi.feeder
+
+actual fun getDefaultLocale(): Locale =
+    if (java.util.Locale.getDefault() == java.util.Locale.JAPAN) {
```

Comment:
How do you think about using `import java.util.Locale` ? 👀

Reply:
@tomoya0x00 
If I use `import java.util.Locale`, it caused an error because the "Locale Class" is the same name.
So I used full path.
If there is a better way, please could you let me know.

Reply:
@Gazyu
Oh, I see. You are right!
Thank you for your explanation 🙇 

We don't need to change this point 👍